repositories.md

```diff
@@ -201,6 +201,14 @@ An OEM SHOULD define a public API for Primaries to use when downloading metadata
 
 Depending on the OEM's requirements, this API MAY require authentication before Primaries are allowed to download updates.  Such a choice affects only how certain the OEM can be that it is communicating with authentic Primaries, and not how resilient ECUs are to a repository compromise. The OEM is free to use any authentication method.
 
+#### Using images from multiple locations
+
+When implementing Uptane, it is often the case that existing software may come from several different locations. It may be tempting to assume that this means that the equivalent Uptane implementation will require multiple different image repositories. However, this is rarely actually necessary, and using multiple image repositories (implemented via [repository mapping metadata as described in TAP-4](https://github.com/theupdateframework/taps/blob/master/tap4.md)) represents a significantly larger effort.
```

Comment:
```suggestion
Uptane implementations may sometimes need to accommodate update systems where existing software comes from several different locations. Implementers may assume that this would mandate the use of multiple different image repositories in any equivalent Uptane implementation. However, this is rarely necessary, and using multiple image repositories (implemented via [repository mapping metadata as described in TAP-4](https://github.com/theupdateframework/taps/blob/master/tap4.md)) would require a significantly larger effort.
```